src/main/java/com/codepine/api/testrail/Request.java

```diff
@@ -129,6 +129,9 @@ public T execute() {
                     try (OutputStream outputStream = new BufferedOutputStream(con.getOutputStream())) {
                         JSON.writerWithView(this.getClass()).writeValue(outputStream, content);
                     }
+                } else {
+                    con.setFixedLengthStreamingMode(0);
+                    con.setDoOutput(true);
```

Comment:
Thanks for the fix. Since this is a common statement between if and else, can you please move it outside?